node/node.go

```diff
@@ -582,6 +621,35 @@ func adjustNftTokenIdentifier(token string, nonce uint64) string {
 	return formattedTokenIdentifier
 }
 
+func (n *Node) getSystemAccount() (state.UserAccountHandler, error) {
+	systemSCAccount, err := n.stateComponents.AccountsAdapterAPI().GetExistingAccount(vmcommon.SystemAccountAddress)
```

Comment:
before this function call you should call something like:
```GO
    rootHash := n.dataComponents.Blockchain().GetCurrentBlockRootHash()
    if len(rootHash) == 0 {
        return nil, ErrEmptyRootHash
    }

    err := n.stateComponents.AccountsAdapterAPI().RecreateTrie(rootHash)
    if err != nil {
        return nil, err
    }
```

Reply:
not relevant anymore